Articles/Blog/2023-04-developer-surveys.md

```diff
@@ -8,114 +8,126 @@
+While Likert scales could
+show a general sentiment, these open-ended responses would be where the heart of developer feedback came in.
 You really can't put people in a check box, nor can you ask them to express a full need or sentiment in one.
 
-Once I had created all of my questions, I put it into a Microsoft Form (what we have to use) and then showed to the RADIUSS team for feedback.
-As an aside, if you are able to use Google Forms I highly recommend this over what we had to use, because I found Microsoft Forms much harder to use.
-We had a few rounds of feedback, and also sent it out for some testing by a set of volunteers.
-As you might expect, many questions were refined, simplified, or entirely removed.
-Our perspective at this time was to do our best, and not worry if it wasn't perfect.
-We would learn from the experience and try differently the next time.
-A few months before the end of 2022, the survey was ready.
+*Q*: Did you test and revise the survey before launching it?
 
-### Launch Away... to Results!
+*Vanessa*: Yes. Once I had created all of my questions, I put them into a Microsoft Form (what we have to use) and then had the RADIUSS team test and give feedback. As an aside, if you are able to use Google Forms I highly recommend this over what we had to use, because I found Microsoft Forms much harder to use. We had a few rounds of feedback, and also sent it out for some testing by a set of volunteers. As you might expect, many questions were refined, simplified, or entirely removed. Our perspective at this time was to do our best, and not worry if it wasn't perfect.
+We would learn from the experience and try differently the next time. A few months before the end of 2022, the survey was ready.
 
-We sent out the survey in the last few months of 2022, making sure to give it fun branding, and advertise on many lists.
+*Q*: How did you launch the survey and gather results?
+
+*Vanessa*: We sent out the survey in the last few months of 2022, making sure to give it fun branding, and advertise it on many lists.
 We sent out a few reminders, and gave people about 2 months to complete it.
-While we don't have a definite number as the survey was sent out to these mailing lists, I suspect the primary demographic we hit was Livermore Computing, and in our Town Hall channel I can see a few hundred people.
-At the end of the day, we had over 100 responses, half of which were self-reported developers, about a quarter of which were staff scientists (that likely worked with code) and the remainder roles like group leads, managers, technical support, or other support staff.
-We had a larger participation than application surveys in previous years, so we were relatively happy.
+While we don't have a definite count of our reach because the survey was sent out on many platforms, I suspect the primary demographic we hit was Livermore Computing and our Town Hall channel.
+I would guesstimate a few hundred people.
```

Comment:
I think we should just remove this - it was my mistake to guess the demographic we hit (we actually have a breakdown, it wasn't shared publicly) I think it sounds OK just removed!

```suggestion
```